src/components/Catalog.js

```diff
@@ -0,0 +1,40 @@
+import React, { Component } from 'react';
+import Movie from './Movie';
+class Catalog extends Component {
+    constructor(){
+        super()
+        this.state = {
+            movieName: ""
+        } }
+    handleInput=(e)=>{
```

Comment:
`handleInput = ({target}) => this.setState({movieName: target.value})`
same thing